contact.html

```diff
@@ -30,50 +24,17 @@
 </p>
 
 <p class="indicates-required">
-  <em><span class="required-label">*</span> Indicates required fields</em>
+  <em>{% include forms/required-label.html %} Indicates required fields</em>
 </p>
```

Comment:
1. Make this an include since it's shared on two pages
2. Probably make this `aria-hidden` since screen readers will explain that certain fields are required anyways